<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputAnswerWindow</class>
 <widget class="QMainWindow" name="inputAnswerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>答案读入</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonChoice">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(177, 177, 177);</string>
    </property>
    <property name="text">
     <string>选择答案图片</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAddress">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>591</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditAnswer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>721</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInputAnswer">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>提取答案</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBarInputAnswer"/>
 </widget>
 <resources/>
 <connections/>
</ui>
